client/components/PetTypesPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react'
+import PetTypesItem from './PetTypesItem'
+
+const PetTypesPage = (props) => {
+    const [petTypes, setPetTypes] = useState([])
+
+    useEffect(() => {
+        const getPetTypes = async () => {

```

Comment:
Let's move the declaration of `getPetTypes` outside the `useEffect` so that the latter doesn't get cluttered.